Sources/Rendering/OpenGL/RenderWindow/index.js

```diff
@@ -362,6 +368,12 @@ function vtkOpenGLRenderWindow(publicAPI, model) {
 
     if (xrPose) {
       const gl = publicAPI.get3DContext();
+
+      if (model.xrSessionIsAR) {
+        gl.canvas.width = model.oldCanvasSize[0];
```

Comment:
Are we guaranteed that `model.oldCanvasSize` is not undefined here? I'm asking because the previous usage of `model.oldCanvasSize` checks against the case where `oldCanvasSize` has not yet been set on the model.

Reply:
Good call, I don't think we can guarantee that `model.oldCanvasSize` is defined. In practice it succeeds for the AR case but there's no harm in adding the extra check just in case.

Reply:
Fixed.